openpype/hosts/openrv/hooks/pre_setup_openrv.py

```diff
@@ -0,0 +1,64 @@
+import os
+import shutil
+import tempfile
+from pathlib import Path
+
+from openpype.lib import PreLaunchHook
+from openpype.hosts.openrv import OPENRV_ROOT_DIR
+from openpype.lib.execute import run_subprocess
+
+
+class PreSetupOpenRV(PreLaunchHook):
+    """Pre-hook for openrv"""
+    app_groups = ["openrv"]
+
+    def execute(self):
+
+        executable = self.application.find_executable()
+        if not executable:
+            self.log.error("Unable to find executable for RV.")
+            return
+
```

Comment:
It would not get to prelaunch hook if executable would not be found.
```suggestion
```

Reply:
The executable path is still needed for the logic below - so I guess it's just the `if` statement that you want removed, not the `find_executable?` Or is there a better way in the prelaunch hook to get the executable path?

Reply:
For what it's worth and for your information:

This PR was originally opened by @ddesmond

I contributed into the PR at the time because I could because I had write permissions to the repo - I currently don't anymore so I am personally unable to contribute to this PR.

I believe @ddesmond is not working on this feature currently meaning that without other contributions from someone with write access this will end up staying as a stale PR.

Reply:
If you need permissions i can give some to you @BigRoy

Reply:
there is also https://github.com/ddesmond/OpenPype/pull/8
You have been added as a colab to my repo @BigRoy , let me know guys if you need more help